xonsh/ptk_shell/updator.py

```diff
@@ -2,13 +2,14 @@
 
 import builtins
 import concurrent.futures
+import logging
```

Comment:
Could you remove logging here too, please?

Reply:
sorry forgot. I will update.

Reply:
Thanks!